english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md

```diff
@@ -0,0 +1,8 @@
+You suddenly travel to Berlin 1989, lying on the road, at night (without wearing cloth!!).
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.